### ISSUE TITLE
Trying to run sheepit using zluda

### ISSUE THREAD

**Issue by Bartixxx32** [vosen/ZLUDA#174]
I think the issue is when some process run subprocess when started with zluda, in this case sheepit detect zluda gpu, but its launching blender and blender seems not be able to detect zluda (propably the blender process is not ran via zluda?)
![image](https://github.com/vosen/ZLUDA/assets/16412925/65895300-7653-4ba1-a23a-026697f8a164)
![image](https://github.com/vosen/ZLUDA/assets/16412925/99422ff1-a8df-4770-8c1d-d050ca4bbb54)

Tried to run as admin, my command im using is 
```zluda.exe -- "C:\Program Files\Eclipse Adoptium\jre-17.0.10.7-hotspot\bin\java.exe" -jar "C:\Users\barto\OneDrive - 0mfx0\Pulpit\animtest\sheepit-client-7.24059.0.jar"```

**Comment by vosen** [comment]
It looks like you are rendering with OptiX. ZLUDA's OptiX does not work with Blender as is not even distributed with ZLUDA

**Comment by Bartixxx32** [comment]
> It looks like you are rendering with OptiX. ZLUDA's OptiX does not work with Blender as is not even distributed with ZLUDA

Yeah, but normally blender works with those if im not wrong it still uses cuda not optix but its named diffriently, or im wrong, but running blender using zluda work well
